model-server-openapi/specifications/model-server.yaml

```diff
@@ -13,481 +13,700 @@ info:
   description: "modelix public API"
   version: "1.0.0"
 servers:
-  - url: '/'
+  - url: '/v2'
     description: model-server
+
+tags:
+  - name: IDs
+  - name: repository management
+  - name: branch management
+  - name: versions
+    description: TODO what is this?
```

Comment:
The `.../branches/...` endpoints operate on the latest version on a branch, which is fine if that is what you want, but sometimes you may want to be more specific.

Reply:
I think we just forgot removing this TODO. That was already clear to us